README.md

```diff
@@ -34,14 +34,14 @@ Current collection
   - Fedora 35
   - ShellCheck, Fedora default (0.7.x)
   - called from 'action.sh'
-  - config: defaults; only enabled for 'test.sh'
+  - config: defaults; 'path' parameter, restricted to 'test.sh'
```

Comment:
It took me a while to realize that this means the path is restricted to 'test.sh' *by default*, it kind of reads as if the only valid value for the 'path' parameter was 'test.sh' (which wouldn't make it a very useful parameter :smile:)

Should config and inputs be two separate sections? I would expect 'config' to be something that customizes the behavior of the tool (disabling some warnings, setting line length etc.) and 'inputs' to specify what the tool should operate on (path to file(s) to check)

Reply:
That's a very good point. Will fix. I plan to do one more pass through all the READMEs after this and doc8 are resolved.